### ISSUE TITLE
add file fail

### ISSUE THREAD

**Issue by pcl5x2008** [cdgriffith/FastFlix#522]
add file show error
![image](https://github.com/cdgriffith/FastFlix/assets/94580176/87a22230-c38d-4a1d-977e-bd19b4a89955)
here is the log and videofile 

```
Traceback (most recent call last):
  File "fastflix\widgets\progress_bar.py", line 94, in run
    task.command(config=self.app.fastflix.config, app=self.app, **task.kwargs)
  File "fastflix\flix.py", line 430, in detect_interlaced
    for line in output.stderr.splitlines():
                ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'splitlines'
2023-09-05 21:10:37,402 - fastflix      ERROR    Exception in 'run': 'NoneType' object has no attribute 'splitlines' - args: (<fastflix.widgets.progress_bar.ProgressBar(0x1a5866539c0, name="ProgressBar") at 0x000001A580486740>,)
Traceback (most recent call last):
  File "reusables\wrappers.py", line 285, in wrapper
  File "fastflix\widgets\progress_bar.py", line 94, in run
    task.command(config=self.app.fastflix.config, app=self.app, **task.kwargs)
  File "fastflix\flix.py", line 430, in detect_interlaced
    for line in output.stderr.splitlines():
                ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'splitlines'
2023-09-05 21:10:37,403 - fastflix      ERROR    Could not properly read the files F:\番剧\23七月\白圣女与黑牧师\[ANi] 白聖女與黑牧師 - 01 [1080P][Baha][WEB-DL][AAC AVC][CHT].mp4
Traceback (most recent call last):
  File "fastflix\widgets\main.py", line 1474, in update_video_info
    ProgressBar(self.app, tasks, hidden=hide_progress)
  File "fastflix\widgets\progress_bar.py", line 63, in __init__
    self.run()
  File "reusables\wrappers.py", line 293, in wrapper
  File "reusables\wrappers.py", line 285, in wrapper
  File "fastflix\widgets\progress_bar.py", line 94, in run
    task.command(config=self.app.fastflix.config, app=self.app, **task.kwargs)
  File "fastflix\flix.py", line 430, in detect_interlaced
    for line in output.stderr.splitlines():
                ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'splitlines'
```

[link removed by admin]

**Comment by phanluchoafptpoly** [comment]
I got the same

**Comment by phanluchoafptpoly** [comment]
open FastFlix setting, and see the path of your ffprobe - try update it to newest one

**Comment by tidypy** [comment]
Research indicates: "try to use the split() method, on a list object instead of a string."
I have the same issue. 
Workarounds tried:  "adding attributes in properties", Running as Admin, Changing file Permissions, Tried various files and filetypes. 

Code change required.

**Comment by cdgriffith** [comment]
Should be fixed as of 5.7.0!
